### PR TITLE
Fix permissions of /tmp/

### PR DESCRIPTION
**Pull request by uraid**
When pulling this docker image on Unraid, and after each update, the `nginx` server fails to load due to permissions issues of `/tmp`:
```
nginx: [emerg] mkdir() "/tmp/nginx" failed (13: Permission denied)
```

This commit chown's the /tmp/ folder to the right permissions so nginx can start correctly

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -263,7 +263,8 @@ RUN apk --update --no-cache add \
   && addgroup -g ${PGID} rtorrent \
   && adduser -D -H -u ${PUID} -G rtorrent -s /bin/sh rtorrent \
   && curl --version \
-  && rm -rf /tmp/*
+  && rm -rf /tmp/* \
+  && chown ${PUID}:${PGID} /tmp/
```
Comment (crazy-max) [inline Dockerfile:267]: Perms `/tmp` are `1777`, which are `drwxrwxrwt` (using `ls`) don't need to be changed. They are already wide open, except that only the owner of a file can remove it (extra `t` bit means for a directory).

That doesn't seem related to this image but your environment with Unraid. Maybe you can find help on this [subreddit](https://www.reddit.com/r/unRAID/).

Comment (uraid) [reply Dockerfile:267]: @crazy-max After checking this out with a clean image of this container, it seems that the perms of `/tmp` are actually `drwxr-xr-x`.

Comment (djerfy) [reply Dockerfile:267]: On Linux Ubuntu with classic Docker install, it's 1777 by default:
```shell
drwxrwxrwt   1 root     root     4.0K Jul 16 21:00 tmp
```
